### DISCUSSION TITLE
SuperTux

### DISCUSSION THREAD

**Discussion by JuliaAgostino** [mayrinck/FOSSonMicrosoftStore#34] [Requests] [answered]
SuperTux hasn't been uploaded to the Microsoft Store yet: https://www.supertux.org/

**Comment by mayrinck** [comment, marked as answer]
As quick as a lightning came @Joshagos, onde he detected we've tried to push Super Tux Kart to the store 😂😂😂

And yes, seems to be a good one to push if Super Tux Kart get approved, they basically follow the same patterns so... Maybe there will be no issues for the other if one gets in... For now, we need to wait.

**Reply by JuliaAgostino** [reply]
> As quick as a lightning came @Joshagos, onde he detected we've tried to push Super Tux Kart to the store 😂😂😂
> 
> And yes, seems to be a good one to push if Super Tux Kart get approved, they basically follow the same patterns so... Maybe there will be no issues for the other if one gets in... For now, we need to wait.

I'm desperate for any developments regarding this project. I'll take anything! 🤣

**Reply by mayrinck** [reply]
That's the spirit! We will progress, time will prove that 🚀

**Reply by JuliaAgostino** [reply]
> That's the spirit! We will progress, time will prove that 🚀

Yeah, I hope this project becomes more popular. It definitely deserves more attention. I'm glad that it's not dead!

Together, we can all make the Microsoft Store a better place!
